src/components/App.js

```diff
@@ -1,28 +1,70 @@
 import React, { Component } from 'react';
-import Autocomplete from './Autocomplete';
+import AutocompleteHooks from './Autocomplete/AutocompleteHooks';
+import CelebrityInfoBox from './CelebrityInfoBox';
 import styles from './App.module.css';
-
-const suggestions = [
-	"The Matrix", "The Matrix 4",
-	"The Matrix Revolutions", "The Matrix Reloaded",
-	"Commando (1985 film)", "The Matrix Revisited",
-	"The Animatrix", "The Transformers: The Movie",
-	"The Living Matrix", "Kung Pow! Enter the Fist"
-];
+import camera from '../camera.png';
 
 class App extends Component {
+	state = {
+		celebrity: {},
+		showInfoBox: false,
+		isLoading: false,
+	};
+
+	fetchCelebrityInfo = (celebrity) => {
+		this.setState({
+			celebrity: celebrity,
+		});
+	};
+
```

Comment:
Check is it right way to lifting state up in child component CelebrityInfoBox, that calls with function inside useCallback hook.